src/balena-init.sql

```diff
@@ -21,6 +21,9 @@ ALTER COLUMN "semver major" SET DEFAULT 0,
 ALTER COLUMN "semver minor" SET DEFAULT 0,
 ALTER COLUMN "semver patch" SET DEFAULT 0;
 
+ALTER TABLE "api key"
+ADD COLUMN IF NOT EXISTS "expiry date" TIMESTAMP NULL;
+
```

Comment:
This should not be necessary - if it is then it's likely due to a bug

Reply:
Added purely for debugging purpose. Removed now.